javascript/0041-first-missing-positive.js

```diff
@@ -0,0 +1,47 @@
+ /**
+   * Time O(n) | Space O(1)
```

Comment:
Suggestion: Fix your empty space and be consistent in your entire file

Reply:
I have fixed the spacing.